content/memory_model.tex

```diff
@@ -52,6 +50,26 @@
 Symmetric data objects are stored on the symmetric heap or in the global/static
 memory section of each \ac{PE}. 
 
+\subsection{Pointers to Symmetric Objects}
+
+Symmetric data objects are referenced in \openshmem operations through the
+local pointer to the desired remotely accessible object.  The address contained
+in this pointer is referred to as a {\em symmetric address}.  Manipulation of
+symmetric addresses, including pointer arithmetic; structure or union member
+access; and array indexing are permitted as long as the resulting local pointer
+remains within the same symmetric allocation or object.  Symmetric addresses
+provided to typed and type-generic \openshmem interfaces must be naturally
+aligned based on their type and any requirements of the underlying
+architecture.  Symmetric addresses are only valid at the \ac{PE} where they
+were generated; using a symmetric address generated by a different \ac{PE}
+results in undefined behavior.
+
+The \FUNC{shmem\_ptr} routine allows the programmer to query a {\em local
+address} to a remotely accessible data object.  The resulting pointer is valid
+for direct load and store access; however, providing this address as an
+argument of an \openshmem routine that requires a symmetric address results in
+undefined behavior.
```

Comment:
@nspark Mention that every symmetric address is also a valid local address (i.e. local address is a superset).